src/Arcus.Templates.WebApi/Controllers/HealthController.cs

```diff
@@ -35,6 +38,10 @@ public HealthController(HealthCheckService healthCheckService)
         [HttpGet(Name = "Health_Get")]
         [ProducesResponseType(typeof(HealthReport), StatusCodes.Status200OK)]
         [ProducesResponseType(typeof(HealthReport), StatusCodes.Status503ServiceUnavailable)]
+#if (ExcludeCorrelation == false && ExcludeCorrelation == false)
+        [SwaggerResponseHeader(200, "RequestId", "string", "The header that will have an unique identifier to distinguish between requests/responses")]
```

Comment:
What about this? This feels more related to what it actually does, no?
```suggestion
        [SwaggerResponseHeader(200, "RequestId", "string", "The header that has a request ID that uniquely identifies this operation call")]
```

Reply:
Same for me.